docs/using.rst

```diff
@@ -193,7 +193,13 @@ Certbot's DNS plugins.
 These plugins are not included in a default Certbot installation and must be
 installed separately. While the DNS plugins cannot currently be used with
 ``certbot-auto``, they are available in many OS package managers and as Docker
-images. Visit https://certbot.eff.org to learn the best way to use the DNS
```

Comment:
I'm not sure if removing the link to the official `certbot` homepage (i.e. where all the basic "how to's" are) is wise. From there, users will be presented with the guide which will either let them use a PPA for example. Or not. Or `certbot-auto` / Docker.